.github/CODEOWNERS

```diff
@@ -84,3 +83,8 @@ ui/components/component-library      @MetaMask/design-system-engineers
 # Slack handle: @accounts-team-devs | Slack channel: #metamask-accounts-team
 
 app/scripts/lib/snap-keyring         @MetaMask/accounts-engineers
+
+# Snaps
+**/snaps/**                          @MetaMask/snaps-devs
+shared/constants/permissions.ts      @MetaMask/snaps-devs
+ui/helpers/utils/permission.js       @MetaMask/snaps-devs
```

Comment:
We might wanna add: https://github.com/MetaMask/metamask-extension/blob/develop/ui/hooks/useTransactionInsights.js

Reply:
Done!